README.md

```diff
@@ -5,16 +5,16 @@
 
 # MatrixNetworks
 This package consists of a collection of network algorithms.
-In short, the major difference between MatrixNetworks.jl and packages like LightGraphs.jl or Graphs.jl is the way graphs are treated.
+In short, the major difference between MatrixNetworks.jl and packages like [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl) or [Graphs.jl](https://github.com/JuliaGraphs/Graphs.jl) is the way graphs are treated.
 
-In [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl), graphs are created through Graph() and DiGraph() which are based on the representation of G as G = (V,E).
+In LightGraphs.jl, graphs are created through `Graph()` and `DiGraph()` which are based on the representation of $G$ as $G=(V,E)$.
```

Comment:
This should probably be `Graphs.jl` now?

Reply:
Yes, you're right.